order.go

```diff
@@ -33,9 +33,15 @@ type ShippingParams struct {
 
 type OrderUpdateParams struct {
 	Params                 `form:"*"`
-	Coupon                 string      `form:"coupon"`
-	SelectedShippingMethod string      `form:"selected_shipping_method"`
-	Status                 OrderStatus `form:"status"`
+	Coupon                 string                     `form:"coupon"`
+	SelectedShippingMethod string                     `form:"selected_shipping_method"`
+	Status                 OrderStatus                `form:"status"`
+	Shipping               *OrderUpdateShippingParams `form:"shipping"`
```

Comment:
This is a bit of a nitpick, but would you mind alphabetizing the fields here? (i.e., put `Shipping` above `Status`.)